plugins/scripts/upload/trns_transform_Genbank_Genome_to_KBaseGenomeAnnotations_GenomeAnnotation.py

```diff
@@ -1677,23 +1683,86 @@ def upload_genome(shock_service_url=None,
             feature_container['type']= feature_type
             feature_container['features'] = features_type_containers_dict[feature_type]
             feature_container['assembly_ref'] = assembly_reference
-            counts_map[feature_type] = len(features_type_containers_dict[feature_type])
-            #Provencance has a 1 MB limit.  We may want to add more like the accessions, but to be safe for now not doing that.
+
+            #Provenance has a 1 MB limit.  We may want to add more like the accessions, but to be safe for now not doing that.
             #provenance_description = "features from upload from %s includes accession(s) : " % (source_name,",".join(locus_name_order))
             feature_container_provenance = [{"script": __file__, "script_ver": "0.1", "description": "features from upload from %s" % (source_name)}]
-
             logger.info("Attempting save of Feature Container %s" % (feature_container_object_name)) 
 
+            do_feature_container_save = True
+
+            container_byte_estimate = len(simplejson.dumps(feature_container))  
```

Comment:
I think it would save memory, and probably be faster to instead count this up yourself, assuming dna_sequence_length counts for the vast majority of the data (if not, you can add to the sum for things like keys and descriptions, aliases, and annotations )
bytes = sum((f['dna_sequence_length'] for f in feature_container.itervalues()))